diff -Nura XdeFactor-ng.orig/src/callbacks.h XdeFactor-ng/src/callbacks.h
--- XdeFactor-ng.orig/src/callbacks.h	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/callbacks.h	2004-03-08 02:48:04.000000000 +0100
@@ -9,7 +9,7 @@
 #include <fcntl.h>
 #include <glib.h>
 #include <dlfcn.h>
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include <pwd.h>
 #include <unistd.h>
 #include <errno.h>
diff -Nura XdeFactor-ng.orig/src/modules/archive_invoices/moduleArchiveInvoices.h XdeFactor-ng/src/modules/archive_invoices/moduleArchiveInvoices.h
--- XdeFactor-ng.orig/src/modules/archive_invoices/moduleArchiveInvoices.h	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/modules/archive_invoices/moduleArchiveInvoices.h	2004-03-08 02:48:04.000000000 +0100
@@ -6,7 +6,7 @@
 #include <stdio.h>
 #include <gdk/gdkkeysyms.h>
 #include <gtk/gtk.h>
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include "../common/defs.h"
 
 void findInvoices (GtkWidget *widget, xdfInvConnModule *conn_mod);
diff -Nura XdeFactor-ng.orig/src/modules/clients/moduleClients.h XdeFactor-ng/src/modules/clients/moduleClients.h
--- XdeFactor-ng.orig/src/modules/clients/moduleClients.h	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/modules/clients/moduleClients.h	2004-03-08 02:48:04.000000000 +0100
@@ -1,5 +1,5 @@
 #include <gtk/gtk.h>
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include <string.h>
 #include <errno.h>
 #include <fcntl.h>
diff -Nura XdeFactor-ng.orig/src/modules/goods/moduleGoods.h XdeFactor-ng/src/modules/goods/moduleGoods.h
--- XdeFactor-ng.orig/src/modules/goods/moduleGoods.h	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/modules/goods/moduleGoods.h	2004-03-08 02:48:04.000000000 +0100
@@ -1,5 +1,5 @@
 #include <gtk/gtk.h>
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
diff -Nura XdeFactor-ng.orig/src/modules/stores/moduleStores.h XdeFactor-ng/src/modules/stores/moduleStores.h
--- XdeFactor-ng.orig/src/modules/stores/moduleStores.h	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/modules/stores/moduleStores.h	2004-03-08 02:48:04.000000000 +0100
@@ -3,7 +3,7 @@
 #include <gtk/gtk.h>
 #include <unistd.h>
 #include <glib.h>
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include <string.h>
 #include "../common/defs.h"
 
diff -Nura XdeFactor-ng.orig/src/tools/xdef.h XdeFactor-ng/src/tools/xdef.h
--- XdeFactor-ng.orig/src/tools/xdef.h	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/tools/xdef.h	2004-03-08 02:48:04.000000000 +0100
@@ -1,5 +1,5 @@
 #include <gtk/gtk.h>
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include <stdlib.h>
 #include <stdio.h>
 
diff -Nura XdeFactor-ng.orig/src/xdfPQexec.c XdeFactor-ng/src/xdfPQexec.c
--- XdeFactor-ng.orig/src/xdfPQexec.c	2004-03-08 02:29:34.000000000 +0100
+++ XdeFactor-ng/src/xdfPQexec.c	2004-03-08 02:48:04.000000000 +0100
@@ -1,4 +1,4 @@
-#include <postgresql/libpq-fe.h>
+#include <libpq-fe.h>
 #include <string.h>
 /*
    funkcja wrapuj±ca PQexec, wy¶wietlaj±ca ewentualne b³êdy w dialogu
